index.html

```diff
@@ -6,22 +6,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Readex+Pro:wght@500&family=Roboto:wght@900&display=swap"
+        rel="stylesheet">
     <title>Devinombre</title>
 </head>
 
 <body>
     <header>
         <h1>Devine!</h1>
-        <p>(Entre 1 et 20)</p>
-        <button>Rejouer</button>
-        <div>?</div>
+        <p class="fourchette">(Entre 1 et 20)</p>
+        <button class="replay">Rejouer</button>
+        <div class="secret">?</div>
     </header>
     <main>
-        <section>
-            <input type="number" />
-            <button>Verifie</button>
+        <section class="gauche">
+            <input type="number" class="userInput" />
+            <button class="checkBtn">Vérifie !</button>
```

Comment:
maybe the button could used the `disabled` keyword at first because at the start the input will not be valid.
reference [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled)

Then when the input become valid you could modify the disabled status from the javascript through the `disabled`attribute of button object something like:

```js
const checkButton = document.querySelector('.check')
checkButton.disabled = false
```

Reply:
done in 66265116ee4680d51b8eaa5853ae7de9c6e90d72